src/index/ExternalVocabulary.cpp

```diff
@@ -74,6 +76,10 @@ void ExternalVocabulary<Comp>::buildFromTextFile(const string& textFileName,
   _size = 0;
   std::string word;
   while (std::getline(infile, word)) {
+    // In the text file we stored the strings with escapend \n and
+    // \\ characters. We now store them in a binary format, where we can use the
+    // `actual` values.
```

Comment:
// The temporary file for the to-be-externalized vocabulary strings is line-based, just like the normal vocabulary file. Therefore, \n and \\ are escaped there. When we read from this file, we have to unescape these.